docs/CONTRIBUTING.md

```diff
@@ -1,21 +1,271 @@
+
+1. Remove the `draft_date` field from the file.
+
+1. Review the feature ID, name, description, and spec fields.
+
+1. Either review, correct, or set the baseline status and browser compatibility data of the feature. This is the most important and difficult step of authoring a feature, which is documented in a separate section. See [Set the status of a feature](#set-the-status-of-a-feature).
+
+### Set the status of a feature
+
+After you've prepared your new feature file with its unique ID, `name`, `description`, and `spec` fields, you must provide the necessary information for this feature's status to be calculated.
+
+A feature's status consists of:
+
+* Whether the feature is considered baseline or not, and which level of baseline it has achieved:
+
+  * Baseline _low_ means that the feature is now available on all browsers listed in the core browser set.
+  * Baseline _high_ means that the feature has been available on all browsers listed in the core browser set for long enough that it's considered _widely_ available.
+
+  For more information about baseline, see [Baseline](./baseline.md).
+
+* If the feature is considered baseline, the dates at which it has achieved the low and high levels.
+
+* The browser support data for this feature, which consists of the version number for each of the browsers that support the feature.
+
+#### Understand where browser support data comes from
+
+The browser support data for your feature is what determines its baseline status. For example, if your feature is supported on Chrome 66, Chrome Android 66, Edge 79, Firefox 76, Firefox Android 79, Safari 12.1, and Safari iOS 12.2, then it's baseline high, because those versions have been released long enough ago.
+
+The browser support data that we use comes from the [browser-compat-data project](https://github.com/mdn/browser-compat-data/) (BCD), which you need to understand before continuing to set the status of your feature.
+
+BCD contains browser support data for individual constituent of web features, such as CSS properties, JavaScript statements, HTML elements and attributes, DOM interfaces, methods, and events.
+For example, BCD contains the list of browsers, and their versions, that support the [`grid-template-rows` CSS property](https://github.com/mdn/browser-compat-data/blob/main/css/properties/grid-template-rows.json), which is a constituent of the [grid](https://github.com/web-platform-dx/web-features/blob/main/features/grid.yml) feature.
+
+We refer to each entry that contains browser support data in BCD as a _BCD key_. In the `grid-template-rows` example, the BCD key is `css.properties.grid-template-rows`.
+
+Features in the web-features project are associated to one or more BCD keys. For example, the grid feature is associated to multiple BCD keys, which together, describe the overall grid feature: `css.properties.display.grid`, `css.properties.display.inline-grid`, `css.properties.grid`, `css.properties.grid-area`, and more.
```

Comment:
Nit: the documentation uses both "associated to" and "associated with". I think the latter is more correct. It would be good to be consistent throughout.